package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.1.1",
   "private": true,
   "engines": {
-    "node": ">=8.9.4 <10.0.0"
+    "node": ">=8.9.4 <=8.12.0"
```

Comment:
minor: maybe we just fix this to whatever we use in Dockerfile (so same as demo/aat/prod,etc)

Reply:
node:8.12.0-slim? Is it slim like compressed JS? We can discuss that more if you like?